src/validators/default-value-validator.ts

```diff
@@ -7,18 +7,22 @@ export const OHRIDefaultFieldValueValidator: FieldValidator = {
     if (codedTypes.includes(field.questionOptions.rendering)) {
       // check whether value exists in answers
       if (!field.questionOptions.answers?.find(answer => answer.concept == value)) {
-        return [{ errCode: 'invalid.defaultValue', errMessage: 'Value not found in coded answers list' }];
+        return [
+          { resultType: 'error', errCode: 'invalid.defaultValue', message: 'Value not found in coded answers list' },
+        ];
       }
     }
     if (field.questionOptions.rendering == 'date') {
       // Check if value is a valid date value
       if (!moment(value, moment.ISO_8601, true).isValid()) {
```

Comment:
This is very restrictive, I know is not part of this PR but you should consider getting this from the client otherwise it will only work for that format.

Reply:
The reason for the restriction is because the default value isn't defined by the client but rather the server side which uses ISO formats; But I agree we need to put the user's locale into consideration